README.md

```diff
@@ -334,9 +334,52 @@ Please see the [RAPIDS Release Selector](https://rapids.ai/start.html) for more
 ## Benchmarking
 cuSignal uses pytest-benchmark to compare performance between CPU and GPU signal processing implementations. To run cuSignal's benchmark suite, **navigate to the topmost python directory ($CUSIGNAL_HOME/python)** and run:
 
-`pytest --benchmark-only`
-
-As with the standard pytest tool, the user can use the `-v` and `-k` flags for verbose mode and to select a specifc benchmark to run. When intrepreting the output, we recommend comparing the _minimum_ execution time reported.
+`pytest --benchmark-enable --benchmark-gpu-disable`
+
+Benchmark are disabled by default in `setup.cfg` providing only test correctness checks.
+
+As with the standard pytest tool, the user can use the `-v` and `-k` flags for verbose mode and to select a specifc benchmark to run. When intrepreting the output, we recommend comparing the _mean_ execution time reported.
+
+To minimize CLI cluster from benchmark results add `--benchmark-columns=LABELS`
```

Comment:
Any description on what this does?

Reply:
Bah, by "this" I mean `--benchmark-columns=LABLELS`. It minimizes colors, but what are we reporting differently?

Reply:
Check out the new wording.